runtime/bcutil/cfreader.c

```diff
@@ -1555,12 +1555,20 @@ checkMethods(J9CfrClassFile* classfile, U_8* segment, U_32 flags)
 			 */
 			if (classfileVersion < BCT_Java7MajorVersionShifted) {
 				method->accessFlags |= CFR_ACC_STATIC;
-				method->accessFlags &= CFR_CLINIT_METHOD_ACCESS_MASK;
-			} else if (J9_ARE_ALL_BITS_SET(method->accessFlags, CFR_ACC_STATIC)) {
-				method->accessFlags &= CFR_CLINIT_METHOD_ACCESS_MASK;
-				/* missing code attribute  detected elsewhere */
+
+			/* if (J2SE_VERSION(vm) >= J2SE_19) { */
```

Comment:
Can you add a comment here to explain why the commented out code is here?  I know there's a valid reason in this case (make it easy to find this location if the 8 vs 9 checks are changed to a compile time decision) and without a comment to remind reviewers, it looks like dead code and might be removed in the future.